docs/usage/shoot_firewall.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Shoot Firewall Configuration
+weight: 15
+---
+
+# Endpoints and Ports of a Shoot Control-Plane
```

Comment:
Why the title is different from the header in the markdown?

Reply:
```
# Shoot firewall

## Endpoints and Ports of a Shoot Control-Plane
```
Makes more sense